content/learn/migration-guides/0.12-to-0.13.md

```diff
@@ -110,7 +110,9 @@ This means users who want to migrate their .meta files will have to add the `inc
     <div class="migration-guide-area-tag">Audio</div>
 </div>
 
-The option to ignore the global volume using `Volume::Absolute` has been removed and `Volume` now stores the volume level directly, removing the need for the `VolumeLevel` struct.
+The option to ignore the global volume using `Volume::Absolute` has been removed and [`Volume`](https://docs.rs/bevy/0.13.0/bevy/audio/struct.Volume.html) now stores the volume level directly, removing the need for the `VolumeLevel` struct.
+
+`Volume::new_absolute` and `Volume::new_relative` were removed. Use `Volume::new(0.5)`.
 
```

Comment:
```suggestion
The option to ignore the global volume using `Volume::Absolute` has been removed and [`Volume`] now stores the volume level directly, removing the need for the `VolumeLevel` struct.

`Volume::new_absolute` and `Volume::new_relative` were removed. Use `Volume::new(0.5)`.

[`Volume`]: https://docs.rs/bevy/0.13.0/bevy/audio/struct.Volume.html
```
I believe this is the current preferred way to link to Rust objects. Mostly a nit pick.